types/loadable__server/loadable__server-tests.tsx

```diff
@@ -104,6 +107,36 @@ const attrFn: AttrFn = (chunk) => {
     const tagsWithAttrFn: string = getStyleTags(attrFn);
 }
 
+// getInlineStyleTags
+{
+    // Should return an array of React elements
+    const elements: Array<React.ReactElement<{}>> = getInlineStyleTags();
+    const elementsWithAttrs: Array<React.ReactElement<{}>> = getInlineStyleTags(
+        attributes
+    );
+    const elementsWithAttrFn: Array<React.ReactElement<{}>> = getInlineStyleTags(
+        attrFn
+    );
```

Comment:
As this should return a promise, those details should reflect this, followign library:
https://github.com/gregberge/loadable-components/blob/f2a6bbd6d7e96780eed8422011942f2f2785c96f/packages/server/src/ChunkExtractor.test.js#L367
https://github.com/gregberge/loadable-components/blob/b0a47face8f84545a3069930ddc2ec572c683d45/website/src/pages/docs/api-loadable-server.mdx#L142-L145

Reply:
f1dc2df